test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Link.Encoded.html

```diff
@@ -39,63 +39,63 @@
 
     <!-- Fallback to static href -->
     <link href="HtmlEncode[[/styles/site.min.css?a=b&c=d]]" rel="stylesheet" data-extra="test" title='"the" title' />
```

Comment:
I mean, I think I saw additional tests using dynamic content. Hope that wasn't just the new `rel='@("stylesheet")'` oddity.